appeals/web/testing/appeals/factory/address.js

```diff
@@ -1,7 +1,7 @@
 import { faker } from '@faker-js/faker';
 import { fake } from '@pins/platform';
 
-/** @typedef {import('@pins/api').Schema.Address} AddressData */
+/** @typedef {import('@pins/applications.api').Schema.Address} AddressData */
```

Comment:
@fabfab-kc some definitions are still imported from Applications - maybe worth copying them over to Appeals